.circleci/config.yml

```diff
@@ -124,13 +124,13 @@ jobs:
       - run:
           name: Package
           command: >-
-            ramp pack -m /workspace/ramp.yml -o
+            python -m RAMP.ramp pack -m /workspace/ramp.yml -o
             /workspace/build/$PACKAGE_NAME.{os}-{architecture}.{semantic_version}.zip
             /workspace/$MODULE_ARTIFACT
       - run:
           name: Package
           command: >-
-            ramp pack -m /workspace/ramp.yml -o
+            python -m RAMP.ramp pack -m /workspace/ramp.yml -o
```

Comment:
Why did you made this change?

Reply:
Yes, `ramp` was failing because it opted to python 3, due to the rmbuilder supporting both python 2 and 3.
The more robust way of invoking `ramp` is `python -m RAMP.ramp` (we use it all over the place).